### ISSUE TITLE
24 accessories not responding after restart of Homebridge on Raspberry PI (Jessie)

### ISSUE THREAD

**Issue by Formpartner** [nfarina/homebridge#1462]
I run Homebridge on a Raspberry Pi latest updates.
I have read related issues but can't find a solution. I have tried loads of stuff. Latest: Erased everything updated raspberry and all on homebridge. I could start it and config all to show except ffmpeg video. This has been working really good for over a year. After restart it only says "24 accessories not responding"
I have 
Philips Hue and 
Nanoleaf
running "parallel" to Homebridge . Both responding and show data next to "24 accessories" not responding (homebridge).

I use plugins (Like before, nothing new):
homebridge-camera-ffmpeg
homebridge-hue (installed but not in config - I use their own)
homebridge-netatmo
homebridge-samsungtv-control
homebridge-telldus

Homebrigde responds: Loaded config.json with 5 accessories and 3 platforms

It runs.

I erase Accessories and persist but no success
I have change name of the bridge but no success
I have updated EVERYTHING

Any advice?

Thanks!
//Johan

**Comment by Formpartner** [comment]
I had turned off Wifi on my Pi since it's also connected with cable. When adding wifi back, is showed again. But still problem if HB restarts config disappear.

**Comment by shidevil** [comment]
could you upload your Config.json and the log of error??

**Comment by Formpartner** [comment]
No, don't work if Raspberry only use cable. I can of course reach it and it look ok in terminal but Home-app can't find it. Actually - now I tried to restart Homebridge and it all works. So knock-knock... I keep it as is for now. Thanks.

**Comment by Formpartner** [comment]
Running Homebridge from Pi. Everything loads and runs as it should. But I can't add it to Home. It show up in HAP tcp as the other Philips Hue and NanoLeaf but my Homebridge don't show in the Home App as the other two... I have now tried most things but no luck. Anyone experience the same thing?

**Comment by NorthernMan54** [comment]
Have you tried changing the user name in your config.json?

**Comment by Formpartner** [comment]
yes

**Comment by idcrook** [comment]
I had this happening to me too.  I had a new phone and some iOS 11 devices on my network, so I do not know what caused it.

I was able to get homebridge + HomeKit working again (with my WeMo and Nest devices) by deleting the Home entirely and starting over from nothing, adding back the bridge, etc...

Had to recreate the Automations and Rooms, and scenes, so it was somewhat annoying, but now _everything_ is working again.

I had noticed that in `~/.homebridge/persist $ cat AccessoryInfo.CC223DE3CE30.json`, the `pairedClients` key was empty before I deleted + re-added everything, and populated after. not sure if that was an indicator or my problem...
